assets/xml/archives/icon_item_static.xml

```diff
@@ -1,194 +1,221 @@
+        <Texture Name="gGreenPotionIconTex" OutName="green_potion_icon" Format="rgba32" Width="32" Height="32" Offset="0x14000" />
+        <Texture Name="gBluePotionIconTex" OutName="blue_potion_icon" Format="rgba32" Width="32" Height="32" Offset="0x15000" />
+        <Texture Name="gBottledFairyIconTex" OutName="bottled_fairy_icon" Format="rgba32" Width="32" Height="32" Offset="0x16000" />
+        <Texture Name="gBottledDekuPrincessIconTex" OutName="bottled_deku_princess_icon" Format="rgba32" Width="32" Height="32" Offset="0x17000" />
+        <Texture Name="gBottledFullMilkIconTex" OutName="bottled_full_milk_icon" Format="rgba32" Width="32" Height="32" Offset="0x18000" />
+        <Texture Name="gBottledHalfMilkIconTex" OutName="bottled_half_milk_icon" Format="rgba32" Width="32" Height="32" Offset="0x19000" />
+        <Texture Name="gBottledFishIconTex" OutName="bottled_fish_icon" Format="rgba32" Width="32" Height="32" Offset="0x1A000" />
+        <Texture Name="gBottledBugIconTex" OutName="bottled_bug_icon" Format="rgba32" Width="32" Height="32" Offset="0x1B000" />
+        <Texture Name="gBottledBlueFireIconTex" OutName="bottled_blue_fire_icon" Format="rgba32" Width="32" Height="32" Offset="0x1C000" />
+        <Texture Name="gBottledPoeIconTex" OutName="bottled_poe_icon" Format="rgba32" Width="32" Height="32" Offset="0x1D000" />
+        <Texture Name="gBottledBigPoeIconTex" OutName="bottled_big_poe_icon" Format="rgba32" Width="32" Height="32" Offset="0x1E000" />
+        <Texture Name="gSpringWaterIconTex" OutName="spring_water_icon" Format="rgba32" Width="32" Height="32" Offset="0x1F000" />
+        <Texture Name="gHotSpringWaterIconTex" OutName="hot_spring_water_icon" Format="rgba32" Width="32" Height="32" Offset="0x20000" />
+        <Texture Name="gBottledZoraEggIconTex" OutName="bottled_zora_egg_icon" Format="rgba32" Width="32" Height="32" Offset="0x21000" />
+        <Texture Name="gBottledGoldDustIconTex" OutName="bottled_gold_dust_icon" Format="rgba32" Width="32" Height="32" Offset="0x22000" />
+        <Texture Name="gBottledMushroomIconTex" OutName="bottled_mushroom_icon" Format="rgba32" Width="32" Height="32" Offset="0x23000" />
+        <Texture Name="gBottledSeahorseIconTex" OutName="bottled_seahorse_icon" Format="rgba32" Width="32" Height="32" Offset="0x24000" />
+        <Texture Name="gChateauRomaniIconTex" OutName="chateau_romani_icon" Format="rgba32" Width="32" Height="32" Offset="0x25000" />
+        <Texture Name="gBottledHylianLoachIconTex" OutName="bottled_hylian_loach_icon" Format="rgba32" Width="32" Height="32" Offset="0x26000" />
+        <Texture Name="gEmptyBottleIcon2Tex" OutName="empty_bottle_icon_2" Format="rgba32" Width="32" Height="32" Offset="0x27000" />
+
+        <!-- Trade Quest icons -->
+        <Texture Name="gMoonTearIconTex" OutName="moon_tear_icon" Format="rgba32" Width="32" Height="32" Offset="0x28000" />
```

Comment:
The name of the item is "Moon's Tear":
![image](https://github.com/zeldaret/mm/assets/12245827/9a41b7f8-6203-412e-9676-26e7a3e262b8)

Did you name it `MoonTear` just to avoid the weirdness around apostrophes?

Reply:
Also changed other instances of moon tear -> moons tear